mongo/session.go

```diff
@@ -89,7 +120,7 @@ func (s *sessionImpl) ID() bsonx.Doc {
 	return s.clientSession.SessionID
 }
 
-// EndSession ends the session.
+// EndSession implements the Session interface.
```

Comment:
This reads wrong, as it's just one method of the interface.  Perhaps "implements a method in the Session interface"

Does this even appear in GoDoc though, since the type is private?

Reply:
This does not appear in GoDoc and we have a bunch of other methods in the code base that have similar documentation, so I'm inclined to leave it as-is.

Reply:
Ack